### ISSUE TITLE
Allow custom labels in installments options

### ISSUE THREAD

**Issue by lluishi93** [Adyen/adyen-web#39]
**Is your feature request related to a problem? Please describe.**
I find difficult to configure (too much magic behind the scenes) and not quite custom the way the installment options are configured, because they use the drop-in format, which in some cases does not result in the expected output.

**Describe the solution you'd like**
Allow custom text in the installments dropdowns, instead of the current format that is done behind the scenes.

**Describe alternatives you've considered**
![image](https://user-images.githubusercontent.com/14786440/85063809-f919dd80-b1aa-11ea-8db5-28f654f1bfba.png)

Allow the developer to, instead of proving a simple array of numbers, each for every installment, allow to provide custom item labels, like:
`installmentOptions: { card: { values: [ { value: 1, label: 'Pay 10000€ at once' }, { value: 2, label: 'Pay 5000€ in two times' } ] } }`

Maybe something like:
```js
const installmentItemsMapper = value => ({
  id: value,
  name: typeof value === 'object' && value.label ? value.label : amount.value ? `${value}x ${getPartialAmount(value)}` : value
})
```

**Additional context**

**Comment by marcperez** [comment]
Hi @lluishi93,

Thank you for your suggestion! We really appreciate your feedback and we will discuss internally what can we do to handle this use case. We'll get back to you once we have an update.

Cheers!

**Comment by lluishi93** [comment]
any news on this one? @marcperez

**Comment by marcperez** [comment]
Hi @lluishi93,

We are currently working on a new feature that should help on this kind of use cases. We'll update you soon.

Cheers!

**Comment by marcperez** [comment]
Hi @lluishi93,

We just released a new feature (#83) that should help you customize the installment option translations. This feature has been included in the v3.10.0 release, which is already available on NPM and will soon be available as a script tag.

As a quick recap of the feature:

The new translation key used in the installment options is `"installmentOption": "%{times}x %{partialValue}"`. This means you can customize the translation including the two variables you already see in the current dropdown.
We also support plurals and specific count translations. An example could be:

```json
const translations = {
  "en-US": {
    "installmentOption": "Pay %{partialValue} at once",
    "installmentOption__plural": "Pay %{partialValue} in %{times} times",
  }
};
```

Cheers!

**Comment by lluishi93** [comment]
Sorry, haven't seen this one! that's awesome, thank you a lot
